src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseEventFieldDefinition.java

```diff
@@ -3,12 +3,11 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.swagger.annotations.ApiModel;
```

Comment:
Could you please move  the imports and take developed  version ?

Reply:
reverted to develop version